packages/@react-spectrum/badge/test/Badge.test.js

```diff
@@ -11,7 +11,7 @@
  */
 
 import {Badge} from '../';
-import CheckmarkCircle from '@spectrum-icons/workflow/src/CheckmarkCircle';
+import CheckmarkCircle from '@spectrum-icons/workflow/CheckmarkCircle';
```

Comment:
corrected all instance of imports like this, we should never import icons from the src file